### PR TITLE
feat(FWS-22): Ability To View All Accounts

### PR DESCRIPTION
**Pull request by AlexeiTimbro**
**JIRA: link to jira ticket**
https://champlainsaintlambert.atlassian.net/browse/FWS-22

**What is the ticket about and why are we making this change?**

This ticket is about being able to view all the user accounts in the system from the admin panel view by the admin of the system. The purpose of this ticket was to facilitate the view of all users in the system for the admins. 

**What are the various changes and what other modules do those changes affect?**
 
-4 types of testing
-backend for the get-all accounts
-Front end for the get-all accounts
- Admin Panel view is completely working and done visually
- Added new nav component for Admin Panel
-Styled both the admin panel view and the admin accounts view

**UI Change:**

Before:
There is no before because there was nothing before. Added the Admin panel link in the nav.
image

After: (Admin Panel View)
image

After clicking the Accounts box:
image

### CODE REVIEW

File: fitwsarah_frontend/src/views/AdminPanelPage/AdminPanel.js [hunk 1]
```diff
@@ -0,0 +1,108 @@
...
+            </Link>
+            <Link className="section" to="/admins">
+              <h2>Admins</h2>
+              <p>Current Total: XX Admins</p>
+              <p>Last Month's Total: XX Admins</p>
+              <span className="more-options">...</span>
+            </Link>
+            <Link className="section" to="/feedback">
+              <h2>Feedback</h2>
+              <p>Current Total: XX Submissions</p>
```
Comment (Eli293) [inline fitwsarah_frontend/src/views/AdminPanelPage/AdminPanel.js:79]: Are the XX placeholders?

File: src/test/java/com/fitwsarah/fitwsarah/accountsubdomain/businesslayer/AccountServiceUnitTest.java [hunk 2]
```diff
@@ -26,10 +47,6 @@ class AccountServiceUnitTest {
     @Mock
     private AccountResponseMapper accountResponseMapper;
 
-    @BeforeEach
-    public void setup() {
-        MockitoAnnotations.openMocks(this);
```
Comment (Abdur-Siddiqui) [inline src/test/java/com/fitwsarah/fitwsarah/accountsubdomain/businesslayer/AccountServiceUnitTest.java:31]: why did you remove the before each method?

**Review by Abdur-Siddiqui — APPROVED** [review]
Looks good, good work

**Review by Eli293 — APPROVED** [review]
Looks fine, good job.

File: src/test/java/com/fitwsarah/fitwsarah/accountsubdomain/presentationlayer/AccountControllerIntegrationTest.java [hunk 3]
```diff
@@ -1,49 +1,90 @@
...
+        Account savedAccount = accountRepository.save(testAccount);
+        testAccountId = savedAccount.getAccountIdentifier().getAccountId();
+    }
+
+    @AfterEach
+    void tearDown() {
+        accountRepository.deleteAll();
+    }
+
+    @Test
```
Comment (Abdur-Siddiqui) [inline src/test/java/com/fitwsarah/fitwsarah/accountsubdomain/presentationlayer/AccountControllerIntegrationTest.java:70]: can you explain how does this work.

Comment (AlexeiTimbro) [reply src/test/java/com/fitwsarah/fitwsarah/accountsubdomain/presentationlayer/AccountControllerIntegrationTest.java:70]: This makes it so that everything a test runs its it deletes the context of the test. So that it doest produce unnescessary data.

File: fitwsarah_frontend/src/views/AdminPanelPage/AdminPanel.js (hunk 1 above)
Comment (AlexeiTimbro) [reply fitwsarah_frontend/src/views/AdminPanelPage/AdminPanel.js:79]: Yes the XX are dummy data used as place holders just to show text for now since we are in developer mode.
